support-frontend/assets/components/thankYou/signIn/signInItems.tsx

```diff
@@ -50,23 +51,31 @@ const hideBeforeTablet = css`
 type SignInBodyCopyProps = {
 	email?: string;
 	csrf: CsrfState;
-	isTier3?: boolean;
+	isTierThree?: boolean;
+	observerPrint?: ObserverPrint;
 };
 
 type CreateSignInUrlResponse = {
 	signInLink: string;
 };
 
-export const signInHeader = (isTier3?: boolean) => {
-	return isTier3
+export const signInHeader = (
+	isTier3?: boolean,
+	observerPrint?: ObserverPrint,
+) => {
+	return observerPrint
+		? 'Sign in to access to your account'
+		: isTier3
```

Comment:
I find nested ternaries difficult to read. Probably early returns would be better
```suggestion
export const signInHeader = (
	isTier3?: boolean,
	observerPrint?: ObserverPrint,
) => {
  if(observerPrint) {
    return 'Sign in to access to your account';
  }
  
    if(isTier3) {
   ...
  }
  
  return default
```

Reply:
Seems a little long-winded for 3 simple cases but can see the readability improvement for more complex items. Ok, updated as suggested.

Reply:
it's longer, it's dumber, but it's a lot better to understand.